packages/DOS/common.dj

```diff
@@ -91,33 +104,59 @@ USE_ARES = 0
 #
 # Enable debug code in libcurl/curl
 #
-USE_DEBUG = 0
+USE_DEBUG = 1
 
 #
 # Enable memory tracking code in libcurl/curl
 #
-USE_CURLDEBUG = 0
+USE_CURLDEBUG = 1
```

Comment:
Again these are all your personal preference I don't see why they should make it upstream